keyboards/westm/westm68/westm68.c

```diff
@@ -16,3 +16,9 @@
 
 
 #include "westm68.h"
+
+void board_init(void) {
+    // Need this to reset first LED upon plugging in PCB
+    rgblight_toggle();
```

Comment:
Not sure that this is the correct way to handle this.  It toggles the status, so could turn off the rgb, if it was on last. (which is the default state)

Reply:
Yeah that's what I thought too, it should just turn off the LEDs. Oddly though it solves the first led being buggy faster upon plugging in, without it the wrong color is displayed for a bit longer. I've since changed it over since it's not a big deal and honestly it's probably for the better!

Reply:
That should not be happening at all...

Reply:
> That should not be happening at all...

I am just as flabbergasted as you are... Wiring wise I've reached out on discord and didn't seem like anyone knows what's going on. For some reason the first LED is getting a stray signal that's causing it to always display the wrong colour at full brightness no matter if LEDs are on or off at the start.

Reply:
Yeah, that's my concern, that this is a hardware issue, and something that should be tracked down, not just worked around.

Reply:
Do you guys have any recommendations? I'm a software person and it's just my friend who's been doing the hardware side of things, and isn't exactly sure how he would fix this problem.

Reply:
Considering this is an ARM board, which runs at 3.3V, and the WS2812s generally run at 5V, there could be a power issue, either on the power supply or the signal. The LEDs should be connected to the 5V supply (ie before the regulator), and the DI pin wired and configured as open drain with a pullup resistor to 5V: https://docs.qmk.fm/#/ws2812_driver?id=push-pull-and-open-drain-configuration

Reply:
Ah okay, thank you for the help we'll try that out and apply the hardware fix to the rest of the boards! In the meantime I've removed the Macropad prototype stuff and the rgb_set from all except the revision 1 on 68 key which I'm using right now, I'll be adding the rev2 once the hardware updates are done. Thanks again for the help!